bower.json

```diff
@@ -1,8 +1,6 @@
 {
   "name": "ember-cli-flash",
   "dependencies": {
-    "ember": "~2.7.0",
-    "ember-cli-shims": "0.1.1",
     "ember-qunit-notifications": "0.1.0",
     "sinonjs": "^1.17.1"
```

Comment:
i'm pretty sure this can be removed as ember-sinon doesn't need this as of 0.5.1